lib/y2lan_restart_common.pm

```diff
@@ -55,14 +67,28 @@ sub initialize_y2lan
     assert_script_run '> journal.log';    # clear journal.log
 }
 
+=head2 open_network_settings
+
+ open_network_settings();
+
+Open yast2 lan overview tab and select first device
+
+=cut
 sub open_network_settings {
     $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'lan');
     accept_warning_network_manager_default;
-    assert_screen 'yast2_lan', 180;       # yast2 lan overview tab
-    send_key 'home';                      # select first device
+    assert_screen 'yast2_lan', 180;    # yast2 lan overview tab
+    send_key 'home';                   # select first device
     wait_still_screen 1, 1;
 }
 
+=head2 close_network_settings
+
+ close_network_settings();
+
+Handle warning popup for firewall and assign it to zone or install yast2 lan packages. Close network settings.
```

Comment:
`s/firewall/firewalld/`
`s/Close network settings./Close network settings checking for return code 0 on serial line./`